src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.SslProvider.cs

```diff
@@ -89,6 +165,7 @@ private static int VerifyCertChain(IntPtr storeCtxPtr, IntPtr arg)
                     using (X509Certificate2 leafCert = new X509Certificate2(leafCertPtr))
                     {
                         success = chain.Build(leafCert);
+                        success &= AddChannelBindingToken(leafCert, arg);
```

Comment:
Since we're not currently setting a channel binding token, do we really want to consider it a connection failure if we can't?